### DISCUSSION TITLE
triple-finger tap for dictionary quick look

### DISCUSSION THREAD

**Discussion by johnnywz00** [toerob/vscode-tads3tools#16] [Ideas]
It is nice to have the Mac Dictionary extension for VSCode, but as often as I look up words (for synonyms, correct usage, whatever), it would be great if the three-finger tap could bring up the quick-look version of the dictionary the way BBEdit does...

**Comment by toerob** [comment]
It sounds nice but I'm not sure how this works. A three tap-gesture would correspond to a triple click on the mouse? Or some kind of tablet for the mac? What would the developer's workstation setup look like?

**Reply by johnnywz00** [reply]
It's actually not a triple tap... it's a tap on the trackpad with three fingers at once. Lots of MacOS shortcuts use it... in the Finder it brings up a QuickLook of a document etc. And in most text programs, the same tap brings up a QuickLook of that word in Dictionary... but if it's complicated, no big deal!

**Reply by toerob** [reply]
Aha, that one! I see.

I think it might be better to do a separate plugin for this, since it seems a bit standalone in its nature. As I understand it, the lookups you are referring to would regular be english words and not tads specific ones? Or am I wrong?

**Reply by johnnywz00** [reply]
Correct, this is not a TADS-specific request... just wasn't sure why VSCode didn't respond to the tap the way most other text programs on Mac do...
